### ISSUE TITLE
Missing requirements for running build

### ISSUE THREAD

**Issue by mdavidsen** [fusetools/fuselibs-public#156]
Clean win10 machine requires additional packages to be able to build and run tests. 

Running the fuse installer (1.0.2) resolves the issue, most likely VCRedist?

```
PS C:\Users\Marius\fuselibs-public\fuselibs-public-master> Stuff\uno --trace doctor
Uno 1.1.0 (build 5395) Windows 10 x64 706c6d5
Found 69 projects, 69 are out of date
Building 1/69: Android.ActivityUtils
FATAL ERROR: Failed to load 'C:\Users\Marius\fuselibs-public\fuselibs-public-master\Stuff\bin\x64\uAsset.dll'. The speci
fied module could not be found (pass --trace for stack trace)
(0.2 s)
```

**Comment by kusma** [comment]
Yeah, this is probably some version of the VCRedist. uAsset.dll is a native C++ library, which will depend on the version of the CRT it was built against. I don't think we have any proper tracking of what version that is, @mortend do you know?

**Comment by mdavidsen** [comment]
After starting over; i've come to the followings steps are needed to run tests (on a clean windows install):
```powershell
choco install git vcredist2013
git clone git://github.com/fusetools/fuselibs-public
cd fuselibs-public
.\stuff\stuff install stuff
.\stuff\uno --trace doctor
.\Stuff\uno --trace test -tDotNet Source
```

If running on a computer without a OpenGL 2.1 context available (e.g. virtual machines), you need an alternative OpenGL implementation (same that is used for the appveyor tests)

```powershell
Invoke-WebRequest http://downloads.fdossena.com/geth.php?r=mesa-latest -OutFile mesa.zip
Expand-Archive mesa.zip mesa
Get-ChildItem -Path Source -Recurse -Include *Test.unoproj | Select-Object -ExpandProperty DirectoryName | Foreach-Object {
        $buildDir = Join-Path $_ build\Test\DotNet
        New-Item -Force -ItemType directory -Path $buildDir | Out-Null
        Copy-Item -Path mesa\opengl32.dll -Destination (Join-Path $buildDir opengl32.dll)
      }
```

This is mostly based on the appveyor-config, so i would suggest that a section on how to build/run tests should be updated. 

Requires Admin-privileges to build it seems currently (no error is thrown during the install/config phase, but you might get some errors related to Network access, because the files originate from a non-trusted internet zone by default.)

**Comment by kusma** [comment]
This was fixed in 014916cb0b16b6f3d3dd9593f1c08dc25f1b4eea.
